server/actions/sendCycleLaunchAnnouncements.js

```diff
@@ -23,8 +23,7 @@ async function _sendAnnouncementToPhase(cycle, phase) {
 function _buildAnnouncement(cycle, projects) {
   let announcement = `🚀  *Cycle ${cycle.cycleNumber} has been launched!*\n`
   if (projects.length > 0) {
-    const projectListString = projects.map(p => `  • #${p.name} - _${p.goal.title}_`).join('\n')
```

Comment:
Since we no longer need any specific project properties, we can be just a little more efficient here.

I'd recommend replacing:
```js
const projects = await Project.filter({cycleId: cycle.id, phaseId: phase.id}).pluck('name', 'goal')
```
with something like:
```js
const numProjects = await Project.filter({cycleId: cycle.id, phaseId: phase.id}).count()
```
...then passing that into `_buildAnnouncement ()` instead of the array of projects.